pkg/feature/serverless/conditions.go

```diff
@@ -0,0 +1,49 @@
+package serverless
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/feature"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/gvr"
+)
+
+func EnsureServerlessAbsent(f *feature.Feature) error {
+	list, err := f.DynamicClient.Resource(gvr.KnativeServing).Namespace("").List(context.TODO(), v1.ListOptions{})
```

Comment:
Do we need to pass empty NS? wouldn't this work?

```suggestion
	list, err := f.DynamicClient.Resource(gvr.KnativeServing).List(context.TODO(), v1.ListOptions{})
```

Reply:
I don't know. This interweb article https://caiorcferreira.github.io/post/the-kubernetes-dynamic-client/ says this:

>  Note that if you are dealing with a namespaced resource then .Namespace(namespace) is obligatory, even if you will use an empty string to list on all namespaces.

After reading that, I didn't even try to omit it.

Reply:
The resource is namespace-scoped.